dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportControllerTest.java

```diff
@@ -390,6 +397,14 @@ void getTrackedEntityByIdNotFound() {
         GET("/tracker/trackedEntities/Hq3Kc6HK4OZ").error(HttpStatus.NOT_FOUND).getMessage());
   }
 
+  @Test
+  void shouldReturnNotFoundWhenGettingASoftDeletedTrackedEntityById() {
+    String uid = softDeletedTrackedEntity.getUid();
+    assertEquals(
```

Comment:
IMHO there is no need to assert on the message. A 404 says it all 😄 Asserting on the exact message always makes a test have to change with the implementation.